factorial_perm_comp.py

```diff
@@ -0,0 +1,79 @@
+# Script Name		: factorial_perm_comp.py
+# Author			: Ebiwari Williams
+# Created			: 20th May 2017
+# Last Modified		: 
+# Version			: 1.0
+
+# Modifications		: 
+
+# Description		: Find Factorial, Permutation and Combination of a Number
+
+
+def factorial(n):	
+	fact = 1
+	while(n >= 1 ): 
+		fact = fact * n 
+		n = n - 1
+
+	return fact
+
+
+def permutation(n,r):
+	return factorial(n)/factorial(n-r)
+
+
+
+def combination(n,r):
+	return permutation(n,r)/factorial(r)
+
+
+def main():
+	print('choose between operator 1,2,3')
+	print('1) Factorial')
+	print('2) Permutation')
+	print('3) Combination')
+
+	operation = input('\n')
+
+	if(operation ==  '1'):
```

Comment:
It would be convenient for the user to enter 1,2 or 3 as input (as per the instructions), rather than '1','2' or '3'. Please check against integer values- 1,2 and 3